programs/x509/cert_write.c

```diff
@@ -559,13 +559,15 @@ int main( int argc, char *argv[] )
     mbedtls_printf( "  . Reading serial number..." );
     fflush( stdout );
 
+#if defined(MBEDTLS_BIGNUM_C)
```

Comment:
This is an example program, it should show good practice and only use the new function, regardless of whether the deprecated one is available or not.

Also, this is where the code for decoding hex to binary would go.